src/Core/Enums/FrameworkId.cs

```diff
@@ -1,16 +1,18 @@
-
+﻿
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Resources;
+
 namespace Axe.Windows.Core.Enums
 {
     public static class FrameworkId
     {
-        public const string DirectUI = "DirectUI";
-        public const string Edge = "MicrosoftEdge";
-        public const string InternetExplorer = "InternetExplorer";
-        public const string WPF = "WPF";
-        public const string WinForm = "WinForm";
-        public const string Win32 = "Win32";
-        public const string XAML = "XAML";
+        public static readonly string DirectUI = DisplayStrings.DirectUI;
+        public static readonly string Edge = DisplayStrings.MicrosoftEdge;
+        public static readonly string InternetExplorer = DisplayStrings.InternetExplorer;
+        public static readonly string WPF = DisplayStrings.WPF;
+        public static readonly string WinForm = DisplayStrings.WinForm;
+        public static readonly string Win32 = DisplayStrings.Win32;
+        public static readonly string XAML = DisplayStrings.XAML;
```

Comment:
These are fixed strings from frameworks. I would not recommend moving them into resources